### ISSUE TITLE
Missing wEth funds in Wormhole

### ISSUE THREAD

**Issue by RafaelFernandez** [solana-labs/oyster#318]
Please help. I used warmhole to transfer wEth from Metamask to Solflare wallet but I did not receive them. I can not see the transaction in wormholebridge.com
Ethereum transaction: [https://etherscan.io/tx/0xb9fef33493c1f485a50c29a54a937f19d02728e580095ad3cd53c6e38b418575] 
Destination address in Solana: 2wWxCKJpdrG1n3puG6g3D2ugj5n6CgZELcWHzfTHaEMZ
Thanks in advance.

**Comment by leoluk** [comment]
Looks like the token account wasn't created: https://explorer.solana.com/address/At8mR9byWp9cKPVn5HK2DNZi4YJUbhvSHkErPUxLMCbX

Please follow yamijuan's advice here to create the missing account: https://github.com/solana-labs/oyster/issues/196#issuecomment-909535731. Let me know once the account exists.

**Comment by RafaelFernandez** [comment]
I have created the account in my wallet:
Wallet address: 2wWxCKJpdrG1n3puG6g3D2ugj5n6CgZELcWHzfTHaEMZ
Account: 8E8EY1ANnS38uakRisVpQTT7mNg9HTzLbwEfLqn4kX9P
Is this what you mean? I sent you an screenshot
![SolanaW](https://user-images.githubusercontent.com/4753571/133440508-c4a8868a-b8b6-42b8-8a80-d3bd5ccd097d.JPG)

**Comment by leoluk** [comment]
No, this is the account that needs to be created: https://explorer.solana.com/address/At8mR9byWp9cKPVn5HK2DNZi4YJUbhvSHkErPUxLMCbX

Please follow the steps in the linked comment

**Comment by RafaelFernandez** [comment]
I followed the steps in the linked comment, but the UI do not asked me to create an account, the only message was asking for approval. May be because a Weth account exists in my wallet as I show you? What can I do to create the account you said?
This is a snapshot of the message, and I aproved in Solana and rejected in Metamask as the comment explained
![wh4](https://user-images.githubusercontent.com/4753571/133694521-ad3803c3-c05e-4f7d-9ed1-9b3732005357.JPG)
.
